00_Base/src/util/middleware/global.exception.handler.ts

```diff
@@ -109,6 +110,15 @@ export class GlobalExceptionHandler implements KoaMiddlewareInterface {
               ),
             );
             break;
+          case ResponseValidationError.name:
+            context.status = HttpStatus.BAD_REQUEST;
```

Comment:
Are you sure you want to return 400 here? OCPI states in `5 Status codes`:

> The transport layer ends after a message is correctly parsed into a (semantically unvalidated) JSON structure. When a message does not contain a valid JSON string, the HTTP error 400 - Bad request MUST be returned.
> If a request is syntactically valid JSON and addresses an existing resource, a HTTP error MUST NOT be returned. Those requests are supposed to have reached the OCPI layer.

Since the validation introduced here is not actually dependent on the client, but rather internal, HTTP 400 doesn't seem correct.